content/en/tracing/setup/nginx.md

```diff
@@ -96,6 +99,34 @@ Complete examples:
 
 After completing this configuration, HTTP requests to NGINX will initiate and propagate Datadog traces, and will appear in the APM UI.
 
+## NGINX Ingress Controller
```

Comment:
The heading should probably mention Kubernetes for a couple of reasons:
- people that aren't aware of kubernetes or ingress controllers will know it's not for them before reading the next paragraph
- people that are aware of it will know it's precisely for them
I think it'll be fine to put `(Kubernetes)` at the end.

Reply:
Sounds good. I think the full name is of the project is "NGINX Ingress Controller for Kubernetes " so I'll use that as the heading.